### PR TITLE
Feature/pop data p2p

### PR DESCRIPTION
**Pull request by Mr-Leshiy**

### CODE REVIEW

File: src/protocol.h [hunk 1]
```diff
@@ -230,6 +230,23 @@ extern const char *GETBLOCKTXN;
...
+  */
+extern const char *ATV;
+
+/**
+  * Contains VTBs
+  */
+extern const char *VTB;
+
+/**
+  * Contains VbkBlocks
```
Comment (Warchant) [inline src/protocol.h]:
```suggestion
  * Contains VbkBlock
```

Comment (Warchant) [reply src/protocol.h]: Single block or multiple blocks?

Comment (Warchant) [reply src/protocol.h]: Same above

File: src/vbk/pop_service_impl.cpp [hunk 2]
```diff
@@ -300,4 +300,86 @@ bool addAllPayloadsToBlockImpl(altintegration::AltTree& tree, const CBlockIndex*
...
+            LogPrint(BCLog::POP, "VeriBlock-PoP: %s ", state.GetPath());
+            return false;
+        }
+
+        return true;
+    }
+
+    if (strCommand == NetMsgType::VBKBLOCK) {
+        LOCK(cs_main);
+        LogPrint(BCLog::POP, "received: VbkBlock\n");
```
Comment (Warchant) [inline src/vbk/pop_service_impl.cpp]: pls refactor these loglines in the final version such that they are printed only with other p2p log lines

File: src/protocol.h (hunk 1 above)
Comment (Mr-Leshiy) [reply src/protocol.h]: single.
fixed

**Review by Warchant — APPROVED** [review]

File: src/vbk/pop_service_impl.cpp (hunk 2 above)
Comment (Mr-Leshiy) [reply src/vbk/pop_service_impl.cpp]: fixed

**Pull request merged**